config/routes.rb

```diff
@@ -32,6 +32,8 @@
   resources :series, only: [:show]
   resources :team_members, path: "team", only: [:index, :show]
 
+  post "newsletter_signup", to: "newsletter_signups#create"
+
```

Comment:
I wasn't sure if there was a specific place in this file for this line so I took a guess. Looking forward to suggestions!

Reply:
This works! Alphabetization is really nice in routes files; but, we're past that here & now.